apps/i18n/common/es_es.json

```diff
@@ -918,11 +918,11 @@
   "lockWhenDone": "Bloquea en cuanto los estudiantes acaben",
   "loggedIn": "Iniciado en: {partner}",
   "loginCardForPrint1": "1.) Ir a [{directLink}]({directLink}) o a la [{joinLink}]({joinLink}) y el tipo en tu código sección letra-6: **{sectionCode}**",
-  "loginCardForPrint2": "2.) Elige tu nombre: **{Nombredeestudiante}**",
+  "loginCardForPrint2": "2.) Elige tu nombre: **{studentName}**",
```

Comment:
Any idea why these are in this sync?

Reply:
oh, woops! I started a sync out of spanish to catch these last week, but got distracted with sprint work and forgot about it. I left the changed files sitting around on the i18n machine, so they got included in the scoop.

Fortunately, them being here won't cause any problems.